backend/uploads/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Having another .gitignore here allows us to include an empty uploads directory in git. Otherwise we get an error when uploading a file if we didn’t manually create an uploads folder first.